### ISSUE TITLE
DualCurrencyField: Use existing Input component instead of native `input`

### ISSUE THREAD

**Issue by escapedcat** [getAlby/lightning-browser-extension#1042]
### Is your feature request related to a problem? 

The component is using `input`:  
https://github.com/getAlby/lightning-browser-extension/pull/1000/files#diff-e341c06a28185f14a891bdb91afc6aa3ecaffb6f2f78dea6afb5e4b7d891d2c7R35

### Describe the solution you'd like

Should be done like here:  
https://github.com/getAlby/lightning-browser-extension/blob/feat/597_currency-support/src/app/components/form/TextField.tsx#L36

### Describe alternatives you've considered

none

### Additional context

Related to: #1000 

### Are you working on this?

No

**Comment by secondl1ght** [comment]
Is this still blocked?

**Comment by escapedcat** [comment]
No, go ahead :)

**Comment by secondl1ght** [comment]
I was looking into this and wanted to check what the benefit of this change was? I was looking at the code and the Input component will return an input element the same as the input element that is currently in the DualCurrencyField component. We would still have to pass all the same props down and it would add an extra step of importing the component. Unless I am missing something?

**Comment by escapedcat** [comment]
Reason for opening this was a feeling of generalization.  
`Input` is used for `TextField` so I thought it would be a benefit using it here as well. Looking closer now I noticed that we moved all functionality from `Input` into this one because we need to combine it with the `fiat` info. Which is displayed "inside of the input".  
In this case we're good here I believe. Thanks for you feedback!   

I guess we could move the `hint` from the `TextField` into the `Input` but that's another task.
